docs/source/quantization.rst

```diff
@@ -1000,6 +1000,116 @@ Custom API Example::
   mq = torch.ao.quantization.quantize_fx.convert_fx(
       mp, convert_custom_config=convert_custom_config_dict)
 
+Deciding Which Quantization Tools to Use for Your Use Case
+----------------------------------------------------------
+
+As we've described above, the quantization codebase equips users with a number of tools and options to quantize their models. These include
+
+* The choice of quantization mode
+
+  * Eager Mode Quantization (Eager Mode)
+  * Fx Graph Mode Quantization (Fx)
+
+* When/How to apply quantization
+
+  * Post Training Quantization (PTQ)
+  * Quantization Aware Training (QAT)
+
+* Quantized Ops Type
```

Comment:
this is partially explained near "There are three types of quantization supported:"